codemods/next/13/remove-get-static-props/src/index.ts

```diff
@@ -41,7 +41,7 @@ export const getQuery = (url: string, params: Record<string, string | string[]>)
   return { ...searchParamsObj, ...params };
 };
 
-export const buildLegacyContext = (headers: ReadonlyHeaders, cookies: ReadonlyRequestCookies, params: Record<string, string | string[]>) => {
+export const buildLegacyCtx = (headers: ReadonlyHeaders, cookies: ReadonlyRequestCookies, params: Record<string, string | string[]>) => {
```

Comment:
Should we update tests after this change?

Reply:
nope, all tests pass